src/components/plugins/edit/EditHandler.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import _ from "lodash";
+
+export default class EditHandler {
+  selectedRecords: Array<Number>;
```

Comment:
[Modification Request] If this stays as an array, might be better for performance if this is always sorted. Another option is to convert it to a data structure with constant lookup time.